examples/src/main/java/com/vesoft/nebula/examples/GraphClientExample.java

```diff
@@ -67,85 +66,88 @@ public static void main(String[] args) {
             return;
         }
 
+        GraphClient client = new GraphClientImpl(args[0], Integer.valueOf(args[1]));
         try {
-            try (GraphClient client = new GraphClientImpl(args[0], Integer.valueOf(args[1]))) {
-                client.connect("user", "password");
-                int code = client.switchSpace(SPACE_NAME);
+            client.connect("user", "password");
+            int code = client.switchSpace(SPACE_NAME);
+            if (code != ErrorCode.SUCCEEDED) {
```

Comment:
p.s. I prefer to put the constant before the variable in a comparison

Reply:
`ErrorCode.SUCCEEDED != code` is better ?